src/script/api/script_object.hpp

```diff
@@ -84,6 +84,11 @@ friend class TestScriptController;
 		static ScriptInstance *active;  ///< The global current active instance.
 	};
 
+	class AutoRestoreAllowDoCommand : private AutoRestoreBackup<bool> {
+	public:
+		AutoRestoreAllowDoCommand(bool allow);
```

Comment:
It looks like `allow` is always passed as `false`, is it needed?

Reply:
Hmm, I could indeed remove the parameter and force `false`.
Requires a new class name then.

Reply:
Maybe something like `DisableDoCommand` or `DisableDoCommandScope`?